lfs.h

```diff
@@ -656,6 +656,20 @@ int lfs_file_rewind(lfs_t *lfs, lfs_file_t *file);
 // Returns the size of the file, or a negative error code on failure.
 lfs_soff_t lfs_file_size(lfs_t *lfs, lfs_file_t *file);
 
+// Move a file handle
+//
+// Littlefs file handles are somewhat expensive to move.
+// Try to avoid needing to move them.
+// This allows moving a file handle from old_file to new_file when needed for abstraction.
+// After this call, old_file is invalid.
+//
+// Returns a negative error code on failure.
```

Comment:
This documentation is good, but if you could reflow the text to match the style of the neighboring documentation:

``` c
// Move a file handle                                                   
//                                                                      
// littlefs's file handles are somewhat expensive to move. Try to avoid 
// needing to move them. This allows moving a file handle from old_file 
// to new_file when needed for abstraction. After this call, old_file is
// invalid.                                                             
//                                                                      
// Returns a negative error code on failure.                            
```